Automattic/wp-calypso
# Expose Hosting Trial: Add trial acknowledge modal

Related to https://github.com/Automattic/dotcom-forge/issues/5097

## Proposed Changes

This PR extracts the trial acknowledge modal from the stepper component to make it reusable. 

* Create a trial acknowledge modal to show acknowledgement outside the stepper 

![image](https://github.com/Automattic/wp-calypso/assets/47489215/584bb8fb-002b-4267-af2a-b0446804a277)


## Testing Instructions



* go to` /setup/new-hosted-site/trialAcknowledge` and verify it still shows in stepper flow


## Pre-merge Checklist
